releng/org.eclipse.tracecompass.target/tracecompass-e4.36.target

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><?pde version="3.8"?><target name="tracecompass-e4.36" sequenceNumber="1">
+<locations>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<repository location="https://download.eclipse.org/justj/jres/21/updates/release/21.0.6/"/>
+<unit id="org.eclipse.justj.openjdk.hotspot.jre.minimal.stripped.feature.group" version="0.0.0"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.cdt.gnu.dsf.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.tm.terminal.control" version="0.0.0"/>
+<unit id="org.eclipse.remote.core" version="0.0.0"/>
+<unit id="org.eclipse.remote.ui" version="0.0.0"/>
+<unit id="org.eclipse.remote.jsch.core" version="0.0.0"/>
+<unit id="org.eclipse.remote.jsch.ui" version="0.0.0"/>
+<repository location="https://download.eclipse.org/tools/cdt/releases/12.0/cdt-12.0.0/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="org.eclipse.swtbot.eclipse.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.swtbot.feature.group" version="0.0.0"/>
+<unit id="org.eclipse.swtbot.generator.feature.feature.group" version="0.0.0"/>
+<repository location="https://download.eclipse.org/technology/swtbot/releases/4.2.1/"/>
+</location>
+<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+<unit id="bcpg" version="0.0.0"/>
+<unit id="bcprov" version="0.0.0"/>
+<unit id="ch.qos.logback.classic" version="0.0.0"/>
+<unit id="com.google.gson" version="0.0.0"/>
+<unit id="com.google.guava" version="33.4.6.jre"/>
+<unit id="jakarta.activation-api" version="1.2.2"/>
+<unit id="jakarta.annotation-api" version="1.3.5"/>
+<unit id="jakarta.el-api" version="5.0.1"/>
+<unit id="jakarta.inject.jakarta.inject-api" version="1.0.5"/>
+<unit id="jakarta.servlet.jsp-api" version="3.1.1"/>
+<unit id="jakarta.xml.bind-api" version="2.3.3"/>
+<unit id="json" version="0.0.0"/>
+<unit id="org.antlr.runtime" version="3.2.0.v20230929-1400"/>
+<unit id="org.antlr.runtime.source" version="3.2.0.v20230929-1400"/>
+<unit id="org.apache.commons.cli" version="0.0.0"/>
+<unit id="org.apache.commons.commons-compress" version="0.0.0"/>
+<unit id="org.apache.commons.commons-io" version="0.0.0"/>
+<unit id="org.apache.commons.jxpath" version="0.0.0"/>
+<unit id="org.apache.commons.lang" version="0.0.0"/>
+<unit id="org.apache.commons.lang3" version="0.0.0"/>
+<unit id="org.apache.xml.serializer" version="0.0.0"/>
+<unit id="org.mortbay.jasper.apache-jsp" version="0.0.0"/>
```

Comment:
Staging target has a fixed version and here the latest will be pulled in. Are both the same? or should we have fixed version here or staging should remove the version?